scripts/install/sonarrv3.sh

```diff
@@ -0,0 +1,179 @@
+#!/bin/bash
+# Sonarr v3 installer
+# Flying sauasges for swizzin 2020
+
+if [[ -f /tmp/.install.lock ]]; then
+  export log="/root/logs/install.log"
+else
+  log="/root/logs/swizzin.log"
+fi
+
+#shellcheck source=sources/functions/ask
+. /etc/swizzin/sources/functions/ask
+
+#Handles existing v2 instances
+_sonarrv2_flow(){
+    v2present=false
+    if [[ -f /install/.sonarr.lock ]]; then
+        v2present=true
+    fi
+    if dpkg -l | grep nzbdrone > /dev/null 2>&1 ; then
+        v2present=true
+    fi
+
+    if [[ $v2present == "true" ]]; then
+        echo "Sonarr v2 is detected."
+        echo "Continuing will migrate your current v2 installation." | tee -a $log
+        echo "You can read more about the migration at https://docs.swizzin.ltd/applications/sonarrv3#migrating-from-v2"
+        echo "An additional copy of the backup will be made into /root/sonarrv2.bak/" | tee -a $log
+        if ! ask "Do you want to continue?" N; then
+            exit 0
+        fi
+
+        if ask "Would you like to trigger a Sonarr-side backup?" Y; then
+            echo "Backing up Sonarr v2" | tee -a $log
+            if [[ -f /install/.nginx ]]; then
+                address="http://127.0.0.1:8989/sonarr/api"
+            else
+                address="http://127.0.0.1:8989/api"
+            fi
+
+            [[ -z $sonarrv2owner ]] && sonarrv2owner=$(cut -d: -f1 < /root/.master.info)
```

Comment:
Was this written before `_get_master_username`?

Reply:
Good catch! I think it was, I'll switch it out tomorrow